acceptance/lib/helper.rb

```diff
@@ -304,7 +304,7 @@ def hup_server(host = master, timeout = 30)
         response = https_request(url, 'GET', cert, key)
       rescue StandardError => e
         expected_errors = [ Errno::ECONNREFUSED, Errno::ECONNRESET, 
-                            OpenSSL::SSL::SSLError ]
+                            Errno::EOFError, OpenSSL::SSL::SSLError ]
```

Comment:
Should this just be `EOFError` and not `Errno::EOFError`?  I don't think `EOFError` is an `Errno`.